.github/workflows/Documenter.yml

```diff
@@ -14,9 +14,8 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
-      matrix:
-        julia-version: [1]
-        os: [ubuntu-latest]
+      julia-version: 1.6
+      os: ubuntu-latest
```

Comment:
I feel like `1` is a more compatible choice since it automatically uses the latest 1.x julia release.

Reply:
I'll do it